modules/Hoot/managers/api.js

```diff
@@ -738,7 +738,14 @@ export default class API {
             .then( resp => {
                 let fileBlob = new Blob( [ resp.data ], { type: 'application/xml' } );
                 saveAs( fileBlob, `changeset_${id}.osc` );
-            });
+            })
+            .catch( err => {
+                console.log(err);
```

Comment:
remove console.log

Reply:
I added this because the promise catch was silently swallowing js errors in addition to xhr errors

Reply:
how about console.error since i see us using that in cases like this?